src/components/AuthMethodButton.tsx

```diff
@@ -81,7 +92,7 @@ export default function AuthMethodButton(props: AuthMethodButtonProps) {
     return () => {
       isSubscribed = false;
     };
-  }, [props.href, acrValue, context.pkce, redirectUri]);
+  }, [props.href, acrValue, context.pkce, redirectUri, action, message]);
```

Comment:
Not sure why the `action` is also added here ?

Reply:
@sgryt You're right, should only be message.